networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -16,6 +16,34 @@ Red Hat supports the Kubernetes NMState Operator in production environments on b
 
 Before you can use NMState with {product-title}, you must install the Kubernetes NMState Operator.
 
+[NOTE]
+====
+The Kubernetes NMState Operator updates the network configuration of the secondary NIC. It cannot update the network configuration of the primary NIC.
+====
+
+{VirtProductName} uses link:https://nmstate.github.io/[`nmstate`] to report on and configure the state of the node network. This makes it possible to modify network policy configuration, such as by creating a Linux bridge on all nodes, by applying a single configuration manifest to the cluster.
+
+Node networking is monitored and updated by the following objects:
+
+`NodeNetworkState`:: Reports the state of the network on that node.
+`NodeNetworkConfigurationPolicy`:: Describes the requested network configuration on nodes. You update the node network configuration, including adding and removing interfaces, by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+`NodeNetworkConfigurationEnactment`:: Reports the network policies enacted upon each node.
+
+{VirtProductName} supports the use of the following nmstate interface types:
```

Comment:
We don't really limit what interface types people can use. Is this list coming from somewhere else?

Reply:
I don't know I just picked up this section with a minor update to make turns out some duplication in this section. I will remove